src/main/java/org/apache/commons/lang3/time/StopWatch.java

```diff
@@ -671,4 +680,368 @@ public void unsplit() {
         splitState = SplitState.UNSPLIT;
     }
 
+    /**
+     * Take the time of the exectution of a given {@linkplain FailableSupplier}.
+     *
+     * <p>
+     * <b>Take the time of the execution of given {@linkplain FailableSupplier}</b>
+     * <pre>{@code
+     * final StopWatch watch = StopWatch.create();
+     *
+     * String result = watch.get(() -> "A");
+     * }</pre>
+     * </p>
+     * <p>
+     *
+     * @param supplier the {@linkplain FailableSupplier} those execution should be measured
+     *
+     * @return the result of the {@linkplain FailableSupplier#get()} operation
```

Comment:
No empty lines is needed between Javadoc tags.

Reply:
Removed